site/docs/v1/tech/apis/api-keys.adoc

```diff
@@ -45,55 +44,75 @@ For security purposes, the [field]#Key manager# setting may be modified only usi
 
-[.api-authentication]
+* link:#create_key_generate_id[Create a API Key with a randomly generated Id]
+* link:#create_key_with_id[Create a API Key with the provided unique Id]
+* link:#copy_key_generate_id[Create a API Key from an existing API Key with a randomly generated Id]
+* link:#copy_key_with_id[Create a API Key from an existing API Key with the provided unique Id]
+
+=== Request
+
+[.api-authentication, id="create_key_generate_id"]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Key with a randomly generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/api-key/#
+--
+
+[.api-authentication, id="create_key_with_id"]
 link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Key with the provided unique Id
 [.endpoint]
 .URI
 --
 [method]#POST# [uri]#/api/api-key/`\{apiKeyId\}`#
 --
 
-=== Request Parameters
+==== Request Parameters
 
 [.api]
 [field]#apiKeyId# [type]#[UUID]# [optional]#Optional# [default]#defaults to a generated UUID#::
 The unique Id of the API Key to create. If not specified a secure random UUID will be generated.
 
 :apikey_create_request:
-include::docs/v1/tech/apis/_apikey-create-update-request-body.adoc[]
+include::docs/v1/tech/apis/_apikey-post-put-request-body.adoc[]
 :apikey_create_request!:
 
-=== Response
-
-The response for this API contains the Key that was updated.
+[.api-authentication, id="copy_key_generate_id"]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Key from an existing Key with a randomly generated Id
 
-:never_search_error:
-include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
-:never_search_error!:
-
-:apikey_create_response:
-include::docs/v1/tech/apis/_apikey-create-update-response-body.adoc[]
-:apikey_create_response!:
+[NOTE.note]
+====
+The API key being copied must not be set as a key manager, if it is, this API will return an error. Also, the resulting new key will have the same permissions as the source key.
+====
 
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/api-key/#
+--
 
-== Create a New API Key From an Existing API Key
+[.api-authentication, id="copy_key_with_id"]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Key from an existing Key with the provided unique Id
 
-This API is used to create a New API Key based on an existing API Key (source key). The API key that is being copied must not be set as a key manager, if it is, this API will return an error. The resulting new key will have the same permissions as the source key.
+[NOTE.note]
+====
+The API key being copied must not be set as a key manager, if it is, this API will return an error. Also, the resulting new key will have the same permissions as the source key.
```

Comment:
The addition of "also" makes it sound like a continuation of the previous statement, and the new key having the same permissions is dependent upon copying a key that is a key manager, which will also throw an error.

```suggestion
The API key being copied must not be set as a key manager, if it is, this API will return an error. The resulting new key will have the same permissions as the source key.
```